### ISSUE TITLE
develop research / thesis / internship options through MS SEC

### ISSUE THREAD

**Issue by jmorwick** [mwsu-csmp/curriculum#48]
depends on #46 and alongside #47

**Comment by jmorwick** [comment]
Yipkei will develop  SEC690 Research Methods, SEC692 Advanced Research, SEC699 Capstone to fulfill this issue, Joey will develop SEC691 Internship in a new issue #74
